src/docs/development/add-to-app/android/project-setup.md

```diff
@@ -97,6 +97,13 @@ This creates a `some/path/my_flutter/` Flutter module project with some Dart
 code to get you started and a `.android/` hidden subfolder that wraps up the
```

Comment:
@xster Is this specific enough to satisfy https://github.com/flutter/flutter/issues/32992?